lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShapeUI.cpp

```diff
@@ -356,7 +366,23 @@ bool ProxyShapeUI::select(MSelectInfo& selectInfo, MSelectionList& selectionList
 
   auto selected = false;
 
```

Comment:
The changes here are what we did originally for VP1 UFE selection. Later we added the VP2 selection (above).